qemu/tests/usb_storage.py

```diff
@@ -193,56 +193,83 @@ def _check_io_size_option(min_io_size="512", opt_io_size="0"):
     vm.verify_alive()
 
     login_timeout = int(params.get("login_timeout", 360))
-    error_context.context("Check usb device information in monitor",
-                          logging.info)
-    output = str(vm.monitor.info("usb"))
-    if "Product QEMU USB MSD" not in output:
-        logging.debug(output)
-        test.fail("Could not find mass storage device")
-
-    error_context.context("Check usb device information in guest",
-                          logging.info)
-    session = _login()
-    output = session.cmd(params["chk_usb_info_cmd"])
-    # No bus specified, default using "usb.0" for "usb-storage"
-    for i in params["chk_usb_info_keyword"].split(","):
-        _verify_string(i, output, [i])
-    session.close()
-    _do_io_test_guest()
-
-    # this part is linux only
-    if params.get("check_serial_option") == "yes":
-        error_context.context("Check usb serial option", logging.info)
-        serial = str(uuid.uuid4())
-        regex_str = r'usb-storage.*?serial = "(.*?)"\s'
-        _check_serial_option(serial, regex_str, serial)
-
-        logging.info("Check this option with some illegal string")
-        logging.info("Set usb serial to a empty string")
-        # An empty string, ""
-        serial = "EMPTY_STRING"
-        regex_str = r'usb-storage.*?serial = (.*?)\s'
-        _check_serial_option(serial, regex_str, '""')
-
-        logging.info("Leave usb serial option blank")
-        serial = "NO_EQUAL_STRING"
-        regex_str = r'usb-storage.*?serial = (.*?)\s'
-        _check_serial_option(serial, regex_str, '"on"')
-
-    if params.get("check_removable_option") == "yes":
-        error_context.context("Check usb removable option", logging.info)
-        removable = "on"
-        expect_str = "Attached SCSI removable disk"
-        _check_removable_option(removable, expect_str)
-
-        removable = "off"
-        expect_str = "Attached SCSI disk"
-        _check_removable_option(removable, expect_str)
-
-    if params.get("check_io_size_option") == "yes":
-        error_context.context("Check usb min/opt io_size option", logging.info)
-        _check_io_size_option("0", "0")
-        # Guest can't recognize correct value which we set now,
-        # So comment these test temporary.
-        # _check_io_size_option("1024", "1024")
-        # _check_io_size_option("4096", "4096")
+    hotplug_unplug = (params["with_hotplug_unplug"] == "yes")
```

Comment:
Please add doc string to line 14 for hotplug/unplug steps.

Reply:
Updated, thanks!